src/main/java/com/sublinks/sublinksapi/utils/DateUtils.java

```diff
@@ -0,0 +1,7 @@
+package com.sublinks.sublinksapi.utils;
+/**
+ * Date Utils
+ */
+public class DateUtils {
+  public static final String FRONT_END_DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSSSSX";
```

Comment:
This format is unique to Lemmy APIs and should be kept within the Lemmy package.

Reply:
Moved DateUtils